lib/core/src/chain_swap.rs

```diff
@@ -380,7 +380,15 @@ impl ChainSwapStateHandler {
 
     async fn claim(&self, chain_swap: &ChainSwap) -> Result<(), PaymentError> {
```

Comment:
Maybe `claim_or_refund` is a better name? Seems to cover both cases, depending on direction.